pfetch

```diff
@@ -246,6 +246,12 @@ get_os() {
             trap '' EXIT
         ;;
 
+        SunOS)
+            read -r distro branch version _ < /etc/release
```

Comment:
Full output of `cat /etc/release`?

Reply:
I was still figuring out how to send it. I guess I'll have to settle for uploading it to a Git repository

Reply:
```
             OpenIndiana Hipster 2019.04 (powered by illumos)
        OpenIndiana Project, part of The Illumos Foundation (C) 2010-2019
                        Use is subject to license terms.
                           Assembled 10 May 2019
```

Reply:
```sh
IFS='(' read -r distro _ < /etc/release
```